sourcemash/api/items.py

```diff
@@ -26,10 +26,23 @@ def get(self, feed_id):
         return {'items': [marshal(item, item_fields) for item in Item.query.filter_by(feed_id=feed_id).all()]}
 
 class ItemAPI(Resource):
+    def __init__(self):
+        self.reqparse = reqparse.RequestParser()
+        self.reqparse.add_argument('vote', type = int, required = True)
+        super(ItemAPI, self).__init__()
 
     def get(self, feed_id, id):
         item = Item.query.get_or_404(id)
         return {'item': marshal(item, item_fields)}
 
+    @login_required
+    def put(self, feed_id, id):
+        ''' Update article vote count '''
+        args = self.reqparse.parse_args()
+        item = Item.query.get_or_404(id)
+        item.totalVotes += args.vote
```

Comment:
I'm not sure how to do this since we need to get the third column from the db.Relationship....